website/templates/project/project_base.mako

```diff
@@ -194,7 +194,8 @@
             urls: {
                 api: nodeApiUrl,
                 web: ${ node['url'] | sjson, n },
-                update: ${ node['update_url'] | sjson, n }
+                update: ${ node['update_url'] | sjson, n },
+                waterbutlerURL: ${node['waterbutler_url']| sjson, n }
```

Comment:
Nit: `waterbutlerURL` is redundant when it's nested. Can be shortened to `waterbutler`

This is consistent with our guidelines [here](https://cosdev.readthedocs.io/en/latest/osf/guidelines.html#responses) (which is about response bodies but is generally applicable).

Reply:
ah, right